### ISSUE TITLE
MOSEI dataset hyperparameters

### ISSUE THREAD

**Issue by JKBox** [yaohungt/Multimodal-Transformer#26]
Could you help to check the hyperparameters to train MOSEI aligned dataset? 
Thanks in advance.

python main.py \
    --aligned \
    --dataset mosei_senti \
    --embed_dropout 0.3 \
    --nlevels 4 \
    --num_heads 8 \
    --num_epochs 20 \
    --batch_size 16

**Comment by yaohungt** [comment]
https://github.com/yaohungt/Multimodal-Transformer/issues/5

You can sweep over more sets of hyperparameters to find the best-performed numbers.

**Comment by JKBox** [comment]
Could you share the hyperparameters that you used? I grid searched, but still 2% lower for accuracy. Thanks in advance.

**Comment by yaohungt** [comment]
The set of the hyperparameters in the paper are what we used. We do not have another one
